pkg/proxy/BUILD

```diff
@@ -26,7 +26,7 @@ filegroup(
     name = "all-srcs",
     srcs = [
         ":package-srcs",
-        "//pkg/proxy/apis:all-srcs",
+        "//pkg/proxy/apis/kubeproxyconfig:all-srcs",
```

Comment:
Why your PR touch `pkg/proxy`?

Reply:
because this was broken, but it was fixed in the mean time in another pr (#55277).

I'll rebase, that should remove that change.